build-support/bin/install_git_hooks.sh

```diff
@@ -52,10 +52,35 @@ function ensure_hook() {
   fi
 }
 
+function ensure_valid_symlink() {
+  HOOK=$1
+  if [[ -L "${HOOK_DIR}/${HOOK}" ]]; then
+    if [[ -e "${HOOK_DIR}/${HOOK}" ]]; then
+      return
+    fi
+
+    read -rp "A ${HOOK} symlink exists, but there is no associated source file. Remove? [Yn]" ok
```

Comment:
Maybe include what it links to, that doesn't exist?

Reply:
Good call - added a target